### DISCUSSION TITLE
Left click stops working randomly Osu!lazer

### DISCUSSION THREAD

**Discussion by goldengun111** [ppy/osu#16086] [Q&A] [answered]
I've been playing multi lobbies on lazer recently and 9/10 left click stop working and I have to restart my game to fix it, I cant even interact with the game menus and settings and changing things like gameplay mouse buttons or full screen to windowed using key binds doesn't help
 version: osu! 2021.1211.0-lazer

**Comment by bdach** [comment]
you haven't even stated what operating system this is on and what you are using (mouse? tablet?) video would also be helpful.

additionally please attach logs from either:
* `%AppData%/osu/logs` (on Windows),
* `~/.local/share/osu/logs` (on Linux and macOS),
* `Android/Data/sh.ppy.osulazer/logs` (on Android).

**Reply by goldengun111** [reply]
Windows 10 
wacom ctl-472 tablet
but left click doesn't work on tablet or on mouse
have restarted game and pc with no difference
video I can't do due to kind of weak pc
[performance.log](https://github.com/ppy/osu/files/7713952/performance.log)
[runtime.log](https://github.com/ppy/osu/files/7713953/runtime.log)
[updater.log](https://github.com/ppy/osu/files/7713954/updater.log)
[database.log](https://github.com/ppy/osu/files/7713955/database.log)
[network.log](https://github.com/ppy/osu/files/7713956/network.log)

**Reply by bdach** [reply]
did you try disabling mouse buttons during gameplay in settings? you may be triggering button presses by touching the tablet with the stylus.

**Reply by goldengun111** [reply]
done that and it didn't help, especially since the issue happens in the menus and not during a map
In the menus I can't interact with thing any buttons like exit or ready and quitting the lobby doesn't change that
only way I've been able to fix it is by alt+F4 and restarting the game

**Reply by bdach** [reply]
the information you've provided is not enough for me to even guess what may be happening sorry. i'd need to see what the inputs look like and what the frame graph is showing on a video.

long shot but do you perhaps have a mouse with a 1000hz polling rate?

**Reply by goldengun111** [reply]
my mouse is 400Hz polling rate so that's not it ill try to see if my pc can handle an OBS recording to get the issue on video but I don't  even know what causes it so it might take a while

**Reply by goldengun111** [reply]
https://imgur.com/jmucdL8 

started working again after clicking on the notifications tab 

spamming the notification button fixes the issue for some reason

**Reply by bdach** [reply]
the game cursor is pink, so this looks like the game thinks the button is stuck held down. still not sure why though. but i don't think the notification icon had to do anything with it, looked like your mouse left the window for a brief moment there.

do you have "high precision mouse" on? in-game tablet drivers enabled? or some other custom tablet drivers? try toggling any of these off and on and see if it helps anything.

**Reply by goldengun111** [reply]
I do have high precision on and I use the hawku drivers rather than the in game ones, as for toggling them ill need issue to happen again so it will take a bit

the cursor left the screen because I started the recording

**Reply by peppy** [reply]
I also saw this yesterday on a stream. It looked to always happen after alt-tabbing from the game - can you confirm whether this is the case?

**Reply by goldengun111** [reply]
It seems like it, usually before the cursor got stuck I alt tabbed to check discord or the like and then went back to the game just to have my cursor stuck, maybe if you are holding left click as you alt tab the game doesn't realize when you let go so it acts like it constantly held down?

**Reply by Theighlin** [reply]
> maybe if you are holding left click as you alt tab the game doesn't realize when you let go so it acts like it constantly held down?

I've just tried and can confirm this is the case. One left click after alt tabbing in then fixes the cursor, so this seems much less serious than the case reported in this discussion
Windows 10: 21H1 
lazer: 2021.1211.0
Reproducible in Fullscreen, Borderless and windowed.

**Reply by goldengun111** [reply]
That's good to know, still it's annoying that it happens in osu!, while in other games it doesn't happen from what I've seen, so it might be worth adding a kind of fix some time in the future, but as this isn't urgent or detrimental to gameplay it probably shouldn't be a priority.

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/issues/16170
